packages/circuits/semaphore.circom

```diff
@@ -4,22 +4,64 @@ include "babyjub.circom";
 include "poseidon.circom";
 include "binary-merkle-root.circom";
 
+// The Semaphore circuit can be divided into 3 main parts.
+// The first part involves the generation of the Semaphore identity,
+// i.e. the public key and its hash, which is called the commitment
+// and is used as a public value.
+// In the second part, it is verified whether the identity commitment is part
```

Comment:
"In the second part, it is verified whether or not the identity commitment is part of the Semaphore group, implemented as a Merkle tree." 

I would rephrase with something like this